scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -982,7 +982,7 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <membersonly> and <wildcard> elements.
   //
   private class PackageElement {
-    private var packageName: String = null
+    private var packageName: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.